readme.md

```diff
@@ -181,10 +198,10 @@ Then, go to [http://localhost:5000/](http://localhost:5000) and you should see a
   })
   ```
 
-* **Question:** The above can be described as middleware. Describe what middleware is in your own words and how it differs from building a route.
+* **Question:** The above can be described as middleware. Describe what middleware is in your own words and how it differs from building a route. Technically route is a type of middleware.  In this case we are using .use to bind to app.
```

Comment:
Middleware is the code/application/module that communicates between two systems, such as between servers and databases. The code above is different than a route because it is always run whenever the application gets to these lines.